jobs/base_model_validator.py

```diff
@@ -185,3 +205,39 @@ def _get_model_id_regex(self):
             str. A regex pattern to be followed by the model id.
         """
         return '^[A-Za-z0-9-_]{1,%s}$' % base_models.ID_LENGTH
+
+
+class BaseSnapshotMetadataModelValidator(beam.PTransform):
+    """Composite beam Transform which returns a pipeline of validation
+    errors.
```

Comment:
Make this comment more specific so that it is understandable for which model is this validator used.

Reply:
Done